docs/TRANSLATIONS.md

```diff
@@ -0,0 +1,152 @@
+
+**Example Path:**
+
+```
+RAWeb/
+└── lang/
+    ├── en_US.json
+    └── pt_BR.json
+```
+
+### Maintaining Keys
+
+- **Do Not Alter Keys:** Each entry in the JSON file consists of a key-value pair. **Only translate the values.** The keys are used by the application to reference the correct text and must remain unchanged.
+
+```json
+{
+  ":count posts in the last 24 hours": ":count posts nas últimas 24 horas"
+}
+```
+
+- **Consistent Formatting:** Ensure that the structure and formatting (such as indentation) of the JSON file remaining consistent with the original.
+
+### Handling Placeholders
+
+Some translation strings contain placeholders (eg: `:count`, `:total`, `:gameTitle`). These placeholders are dynamically replaced by the application at runtime.
+
+- **Keep Placeholders Intact:** Do not translate or modify placeholders. Ensure they remain exactly as they are in the original key.
+
+**Example:**
+
+```json
+{
+  "Next :count": "Próximo :count"
+}
+```
+
+- **Proper Placement:** Ensure placeholders are positioned correctly within the translated string to maintain the intended meaning. You can move placeholders as appropriate for your target language.
+
+## Submitting Your Changes
+
+1. **Locate the Correct JSON File:**
+
+Navigate to the _lang_ directory and open the JSON file corresponding to your target language. If the file does not exist, you can create a new one following the existing naming convention.
+
+**Example:**
+
+```
+lang/
+└── pt_BR.json
+```
+
+2. **Translate the Values:**
+
+- Open the JSON file in a text editor or an IDE.
+- Translate each value while keeping the keys unchanged.
+- Ensure that the JSON syntax remains valid (commas, quotes, brackets).
+
+3. **Validate JSON Syntax:**
+
+Use a JSON validator to ensure there are no syntax errors. Tools like [JSONLint](https://jsonlint.com/) can be helpful.
+
+4. **Commit Your Changes:**
+
+```shell
+git add .
+git commit -m "chore: update translations for pt_BR"
+```
+
+5. **Push to Your Fork:**
+
+```shell
+git push --set-upstream origin translate/pt_BR
+```
+
+6. **Open a Pull Request:**
+
+- Navigate to the [RAWeb repository](https://github.com/RetroAchievements/RAWeb) repository on GitHub.
+- Click the **Compare & pull request** button.
+- Provide a clear title and description for your pull request.
+- Submit the pull request for review.
+
+## Review Process
+
+Once your pull request is submitted:
+
+1. **Automated Checks:**
+
+The repository's CI/CD pipeline will run automated tests to ensure there are no syntax errors in your JSON files.
+
+2. **Manual Review:**
+
+A maintainer will review your translations for adherence to these guidelines.
+
+3. **Feedback:**
+
+You may receive feedback or requests for revisions. Address any feedback promptly to facilitate merging your changes.
+
+4. **Merging:**
+
+Once approved, your translations will be merged into the `master` branch and deployed with the next site release.
```

Comment:
Would it be worth noting how to delete the branch after it gets merged?

Reply:
Added in latest.